docs/browse.md

```diff
@@ -0,0 +1,37 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Browse
+
+<FeatureAvailability/>
+
+Browse is one of the primary entrypoints for discovering different Datasets, Dashboards, Charts and other DataHub Entities.
+
+Browsing is useful for finding data based on a hierarchical structure. A user might want to find all of their Datasets of a particular platform
+under a particular naming prefix for an organization, for example finding all Snowflake Datasets under the long_tail_companion organization organized under the analytics prefix.
```

Comment:
Let's be a bit more explicit here --

> Browsing is useful for finding data entities based on a hierarchical structure set in the source system. Generally speaking, that hierarchy will contain the following levels:
> 
> * Entity Type (Dataset, Dashboard, Chart, etc.)
> * Environment (prod vs. dev)
> * Platform Type (Snowflake, dbt, Looker, etc.)
> * Container (Warehouse, Schema, Folder, etc.)
> * Entity Name
> 
> For example, a user can easily browse for Datasets within the PROD Snowflake environment, the  `long_tail_companions` warehouse, and the  `analytics` schema: 

![datahub_browse](https://user-images.githubusercontent.com/15873986/190827072-492021d9-f5c3-4a79-acd9-e0e0a616c9b5.gif)

Reply:
If you can send me this video or add it to the static assets I can add the link to it as well